pkg/actions/commands.go

```diff
@@ -166,6 +166,31 @@ func Commands() {
 						},
 					},
 				},
+				{
+					Name:    "list",
+					Aliases: []string{"ls"},
+					Usage:   "list projects",
+					Flags: []cli.Flag{
+						cli.StringFlag{Name: "conid", Value: "local", Usage: "The connection id of the remote deployment to use", Required: false},
+					},
+					Action: func(c *cli.Context) error {
+						ProjectList(c)
+						return nil
+					},
+				},
+				{
+					Name:    "get",
+					Aliases: []string{},
```

Comment:
If there is no aliases for the command this you dont need to have this line

Reply:
I've removed this and removed all other blank aliases.